code/game/objects/items/toys.dm

```diff
@@ -332,7 +334,7 @@
 
 /obj/item/toy/katana
 	name = "replica katana"
-	desc = "Woefully underpowered in D20. Almost has a sharp edge."
+	desc = "Woefully underpowered in D20."
```

Comment:
Unmarked cit change here. Modularize the change

Reply:
not really worth the entire file addition